examples/external-dns/terraform.tfvars.example

```diff
@@ -0,0 +1,9 @@
+dns_create_record    = "true"
+node_count           = "value must be either 1 or 2"
+network              = "projects/xxxxxxx/global/networks/xxxxxxx"
+subnetwork           = "xxxxxxxxx"
+namespace            = "user-friendly-name"
+fqdn                 = "tfe.my.domain.com"
+ssl_certificate_name = "certificate-name-xxxxxxxxxxxxxxxxxxxx"
+dns_zone_name        = "my.domain.com"
+license_file         = "/files/license.rli"
```

Comment:
I do not see this complete list of variables in the variables.tf file for this example, nor in the main.tf.

Reply:
removed node_count, network, subnetwork and ssl_certificate_name